### ISSUE TITLE
get documentation from gir

### ISSUE THREAD

**Issue by benwaffle** [benwaffle/vala-language-server#38]
most of the time, the documentation is in the .gir file, not the .vapi

**Comment by Prince781** [comment]
Is it *ever* in the vapi?

**Comment by Prince781** [comment]
I took a look at GirParser. If you add GIR files to the CodeContext then documentation views just work, since the GirParser populates comment attribute of symbols. 
The problem is that GIR files and VAPIs are mutually exclusive, which creates problems when one is requested by the project configuration. The other problem is that glib-2.0.vapi is required by default and you cannot omit it at all nor substitute GLib-2.0.gir for it. And even if you could load both files, you'd just have two separate symbol trees for each file.

Basically, what I need is something that can match symbols from a .vapi file to symbols found in a parsed GIR file. This would require probably just a few modifications to GirParser in the right place.
